src/constitutive/TACSBladeStiffenedShellConstitutive.cpp

```diff
@@ -2780,17 +2780,17 @@ TacsScalar TACSBladeStiffenedShellConstitutive::bucklingEnvelopeSens(
   TacsScalar root = sqrt(N1Term * N1Term + 4.0 * N12Term * N12Term);
```

Comment:
Since you're adding 1e-13 to the root in all the equations below, you could just add it here instead. Like `TacsScalar root = max(sqrt(N1Term * N1Term + 4.0 * N12Term * N12Term), 1e-13);`